_posts/2020-06-14-Emendations to Source Editions on IrishGen.md

```diff
@@ -0,0 +1,51 @@
+
+> Cu Ulad m Domnaill m Gillai Ultain m Oengusa m Cainelbain m Mael Chróin m Domnaill m Con Rai m Oengusa m Fheradaig Araide Bibre in cáinte de Mumnechaib ba sé ba rectaire do Chormac .h. Chuind. Cairech a ben. Is i ro anacht Fiacha mc Oengusa. inde dicitur Fiacha Araide a quo Dal Araide. m Mael Duin Dergainig m Colmáin m Aeda m Libuir m Dalleni m Ennai m Loegaire m Neill Noigiallaig.
+> > Cú Ulad, son of Domnall, son of Gilla Ultain, son of Óengus, son of Caindelbán, son of Mael Chróin, son of Domnall, son of Cú Rai, son of Óengus, son of Feradach (Araide Bibre. The satirist from among the Munstermen. He was steward to Cormac, grandson of Conn. Cairech was his wife. It was she who cared for Fiacha, son of Óengus. Thence he is called Fiacha Araide, whence are called the Dál nAraide), son of Máel Duin Dergainig, son of Colmán, son of Áed, son of Libur, son of Dallene, son of Enna, son of Lóegaire, son of Níall Noígiallach.
+
+This is impossible. Cormac, grandson of Conn, is Cormac mac Airt. He is Níall Noígiallach's legendary ancestor, placed five generations previous to him in the genealogies. Cormac's steward cannot be Níall's descendant, especially not by nine generations.
+
+## The Edition
+The passage quoted above is [the entry on Cenél Lóegaire](https://celt.ucc.ie//published/G800011F/index.html) from the edition of the Book of Leinster as digitised on CELT (the translation is my own). In [the printed edition](https://www.vanhamel.nl/codecs/Best,_et_al._1954-1983) (p. 1467), however, it is made clear that passage contains a section that has been added in the margin of the manuscript page. I have emphasised the inserted material in bold; it is presented in a footnote in the printed edition but placed after "m Fheradaig".
+
+> Cu Ulad m Domnaill m Gillai Ultain m Oengusa m Cainelbain m Mael Chróin m Domnaill m Con Rai m Oengusa m Fheradaig **Araide Bibre in cáinte de Mumnechaib ba sé ba rectaire do Chormac .h. Chuind. Cairech a ben. Is i ro anacht Fiacha mc Oengusa. inde dicitur Fiacha Araide a quo Dal Araide.** m Mael Duin Dergainig m Colmáin m Aeda m Libuir m Dalleni m Ennai m Loegaire m Neill Noigiallaig.
+> > Cú Ulad, son of Domnall, son of Gilla Ultain, son of Óengus, son of Caindelbán, son of Mael Chróin, son of Domnall, son of Cú Rai, son of Óengus, son of Feradach **Araide Bibre (The satirist from among the Munstermen. He was steward to Cormac, grandson of Conn. Cairech was his wife. It was she who cared for Fiacha, son of Óengus. Thence he is called Fiacha Araide, whence are called the Dál nAraide)**, son of Máel Duin Dergainig, son of Colmán, son of Áed, son of Libur, son of Dallene, son of Enna, son of Lóegaire, son of Níall Noígiallach.
+
+## The Manuscript
+Turning to the manuscript, the insertion (now difficult to read) appears in the lower margin, with a special symbol (an upward pointing angle containing three dots) that will mark its point of insertion in the main text:
+
+!["blogpost2_img#2"](insertion_point.jpg)
```

Comment:
One way to get around the need for permission is to just link to the file.

Reply:
We need the images of the specific bits of the page I am talking about. It isn't reasonably to expect non-specialists to find them by themselves.

Reply:
OK, That's fine then :).